examples/uganda/uganda/space.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import gzip
+import uuid
+
+import geopandas as gpd
+import mesa
+import rasterio as rio
+
+from mesa_geo.geoagent import GeoAgent
+from mesa_geo.geospace import GeoSpace
+from mesa_geo.raster_layers import Cell, RasterLayer
+
+
+class UgandaCell(Cell):
+    population: float | None
+
+    def __init__(
+        self,
+        pos: mesa.space.Coordinate | None = None,
+        indices: mesa.space.Coordinate | None = None,
+    ):
+        super().__init__(pos, indices)
+        self.population = None
+
+    def step(self):
+        pass
+
+
+class Lake(GeoAgent):
+    pass
+
+
+class UgandaArea(GeoSpace):
+    def __init__(self, crs):
+        super().__init__(crs=crs)
+
+    def load_data(self, population_gzip_file, lake_zip_file, world_zip_file):
+        world_size = gpd.GeoDataFrame.from_file(world_zip_file)
+        with gzip.open(population_gzip_file, "rb") as population_file:
```

Comment:
We might want to package those with statements into a function for easy reuse.

Reply:
Not in this PR, but in the future.

Reply:
Sure, have created an issue for it: https://github.com/projectmesa/mesa-geo/issues/91.